packages/corvid-local-site/src/codeCompletion.js

```diff
@@ -18,13 +19,21 @@ let corvidTypes,
 try {
   corvidTypes = require("corvid-types");
 
+  const SHARED_COMPILER_OPTIONS = {
+    composite: true,
+    noEmit: false,
```

Comment:
@agankarin 
Any particularly reason we currently have `noEmit: true` in the base configs in `corvid-types`?
I think if the user does not have any TS files then this flag will have no effect anyways.